app/src/main/java/pjwstk/receipemate/app/view/category/CategoryImageView.java

```diff
@@ -0,0 +1,17 @@
+package pjwstk.receipemate.app.view.category;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class CategoryImageView {
+    private String href;
+
+    private String alt;
+
+    private String title;
+
```

Comment:
tutaj tak samo, zastanawiam się też czy nie usunąć @AllArgsConsturctor i ustawiać tą wartość setterem zamiast w konstruktorze, rozjaśni to kod i nie trzeba będzie zgadywać jakie pole musimy teraz wypełnić